src/main.py

```diff
@@ -120,6 +120,25 @@ def caw():
     # not logged in, yell at them to log in
     return redirect('/')
 
+@app.route('/give_fry', methods=['POST'])
+def fries_for_crobs():
+    post_id = ''
+    if 'user_id' in session and session['user_id']:
+        # logged in
+        if 'postiboi' in request.form and request.form['postiboi']:
+            post_id = request.form.get('postiboi')
+            fries = request.form.get('frybois')
+            print("post id is " + str(post_id))
+            print(type(post_id))
+            print("current fry count is " + str(fries))
+            print(type(fries))
+            with app.app_context():
+                db = get_db()
+                cur = db.cursor()
+                cur.execute('''UPDATE posts SET fries = ? WHERE id = ?''', (str(int(fries) + 1), post_id))
```

Comment:
```suggestion
                cur.execute('''UPDATE posts SET fries = ? WHERE id = ?''', (int(fries) + 1, post_id))
```
this should be fine?